flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/calls/ScalarOperatorGens.scala

```diff
@@ -1423,7 +1423,7 @@ object ScalarOperatorGens {
          |    $resultTerm = $nullTerm ? $defaultValue : $arrayGet;
          |    break;
          |  default:
-         |    throw new RuntimeException("Array has more than one element.");
+         |    throw new org.apache.flink.table.api.TableRuntimeException("Array has more than one element.");
```

Comment:
can we have a test for this?

i guess something like 
```sql
SELECT element(array[1, 2, 3]);
```

Reply:
Idk if you saw my comment [here](https://github.com/apache/flink/pull/26719#discussion_r2178469627), but when I tried adding a test like that, it would break the testing framework

Reply:
commented